app/uk/gov/hmrc/fhregistrationfrontend/controllers/Application.scala

```diff
@@ -105,10 +105,10 @@ class Application @Inject()(
   }
 
   def confirmDelete = userAction.async { implicit request ⇒
-      save4LaterService.fetchLastUpdateTime(request.userId) flatMap {
-        case Some(savedDate) ⇒ Future successful Ok(confirm_delete(new DateTime(savedDate)))
-        case None            ⇒ Future successful ServiceUnavailable
-      }
```

Comment:
Avoid including formatting changes with your PR. If you want to do formatting changes raise a separate PR for this.